.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
```

Comment:
this can be in .gitignore right?

Reply:
It's not a local file per se, as it contains project wide settings. But it's also not vital to make anything run, I think visual studio just generates it automatically.